app/messages.xlf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Base file for translations so you can easily make a messages.??.xlf from a common template instead of taking an existing translation file and modifying it.